crypto/evp/evp_locl.h

```diff
@@ -60,6 +60,36 @@ struct evp_kdf_ctx_st {
     EVP_KDF_IMPL *impl;          /* Algorithm-specific data */
 } /* EVP_KDF_CTX */ ;
 
+struct evp_keymgmt_st {
+    int id;                      /* libcrypto internal */
+
+    const char *name;
+    OSSL_PROVIDER *prov;
+    CRYPTO_REF_COUNT refcnt;
+    CRYPTO_RWLOCK *lock;
+
+    /* Domain parameter routines */
+    OSSL_OP_keymgmt_importdomain_fn *importdomain;
```

Comment:
I find the use of "domain" here surprising. Elsewhere when talking about parameters we use the abbreviation "param"

Reply:
From reading up on the nomenclature for dh and dss, I see "domain parameters" used everywhere.  That's the basis for my choice, and also a wish to not have that too easily mixed up with the standard `set_params` / `get_params` calls.

Reply:
(standard `set_params` / `get_params` calls don't create objects...)

Reply:
I suppose "domainparams" might work, although it is a little long.

Reply:
Or "algparams"

Reply:
"domparams"?

Reply:
domparams is fine by me.

Reply:
Ok, will fix

Reply:
This still seems to be outstanding

Reply:
Ah, right

Reply:
Done